mdf_connect_server/automate/globus_automate_flow.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import globus_automate_client
+import mdf_toolbox
+
+from mdf_connect_server.automate.flow_action import FlowAction
+
+
+class GlobusAutomateFlow:
+    def __init__(self, native_app_id: str, flow_id: str = None, flow_scope: str = None):
+        self.flows_client = globus_automate_client.create_flows_client(native_app_id)
+        self.flow_id = flow_id
+        self.flow_scope = flow_scope
+
+    def __str__(self):
+        return f'Globus Automate Flow: id={self.flow_id}, scope={self.flow_scope}'
```

Comment:
F-strings aren't Python 3.6-compatible, and Connect Server has been delayed migrating from 3.6 due to package requirements (which have likely been resolved now, but that's a task for another time). Can you change this to a `.format()` instead?